src/forklift/cli.py

```diff
@@ -18,7 +18,7 @@
 from email.mime.text import MIMEText
 from email.mime.multipart import MIMEMultipart
 from git import Repo
-from importlib import import_module
+from imp import load_source
```

Comment:
This allow for loading a module using the full path to the file thus guaranteeing uniqueness in our instance.